core/chains/evm/txmgr/models.go

```diff
@@ -42,6 +43,10 @@ type TransmitCheckerSpec struct {
 	// VRFCoordinatorAddress is the address of the VRF coordinator that should be used to perform
 	// VRF transmit checks. This should be set iff CheckerType is TransmitCheckerTypeVRFV2.
 	VRFCoordinatorAddress common.Address
+
+	// VRFRequestBlockNumber is the block number in which the provided VRF request has been made.
+	// This should be set iff CheckerType is TransmitCheckerTypeVRFV2.
+	VRFRequestBlockNumber *big.Int
```

Comment:
Do we need the same bugfix in v1 too? Definitely doesn't have to be in this PR / ticket though

Reply:
I think in theory similar thing could occur for V1, but we haven't seen it yet I don't think.